### PR TITLE
pod: added network unit tests

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/pod/network_test.go [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+		if testCase.expectedNil {
+			assert.Nil(t, annotation)
+		} else {
+			assert.NotNil(t, annotation)
+		}
+	}
+}
+
+func TestPodNetworkStaticIPBondAnnotationWithInterface(t *testing.T) {
+	testCases := []struct {
```
Comment (klaskosk) [inline pkg/pod/network_test.go:355]: There's no test for when ipAddrBond is empty. There is a test for when it is invalid but not empty

**Review by klaskosk — COMMENTED** [review]
a couple of minor comments

**Review by klaskosk — APPROVED** [review]
/lgtm

**Review by evgenLevin — APPROVED** [review]
lgtm

**Pull request merged**